content/en/catalyst/fund9/challenges/the-great-migration-eth/campagin-brief.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "The Great Migration (from Ethereum)"
+linkTitle: "The Great Migration (from Ethereum)"
+type: docs
+hide_feedback: true
+description: "How might we make it easy and fast for dApps and other projects to expand or completely move from Ethereum to Cardano?"
+---
+<!-- <img src="https://cardano.ideascale.com/community-library/accounts/93/936143/Public/21-Community-Advisor-Improvements-dbdbe2.png" style="width:500px;height500px"> -->
+
+Fund Size: **500,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/c/idea/405419) and [browse ideas]().
```

Comment:
Do we need to change the campaign link to this link, 'https://cardano.ideascale.com/c/campaigns/26593/about'?